DTS-STN/Alpha-Site
# Update project page

# Description

Update project page to read data from CMS. This pr is similar to #487, but for project page.

[react-markdown](https://www.npmjs.com/package/react-markdown) package added to read markdown text, make sure to `yarn install` before run the project

## Acceptance Criteria

Read content from cms

## Test Instructions

1. run the project locally `yarn dev`
2. go to project page
3. check if the page content displays properly
4. click buttons to see if navigation works properly

## Checklist
